plugins/flespiConnector.ts

```diff
@@ -3,13 +3,49 @@ import { Context, Plugin } from "@nuxt/types";
 import Vue from "vue";
 import { Inject } from "@nuxt/types/app";
 
+/// Models
+/**
+ * Position interface to get position data from a tracker
+ */
```

Comment:
We should look if we can export interfaces, types, etc... into the types folder

Reply:
Should this be a different issue?

Reply:
Yep, we can split this off into a separate issue